runtime/kusama/src/weights/pallet_elections_phragmen.rs

```diff
@@ -116,75 +116,67 @@ impl<T: frame_system::Config> pallet_elections_phragmen::WeightInfo for WeightIn
 	// Storage: Council Proposals (r:1 w:0)
 	// Storage: Council Members (r:0 w:1)
 	fn renounce_candidacy_members() -> Weight {
-		(44_936_000 as Weight)
+		(45_187_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(4 as Weight))
 			.saturating_add(T::DbWeight::get().writes(4 as Weight))
 	}
 	// Storage: PhragmenElection RunnersUp (r:1 w:1)
 	fn renounce_candidacy_runners_up() -> Weight {
-		(32_220_000 as Weight)
+		(32_879_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
 	// Storage: Benchmark Override (r:0 w:0)
 	fn remove_member_without_replacement() -> Weight {
 		(2_000_000_000_000 as Weight)
 	}
-	// Storage: PhragmenElection RunnersUp (r:1 w:1)
 	// Storage: PhragmenElection Members (r:1 w:1)
 	// Storage: System Account (r:1 w:1)
+	// Storage: PhragmenElection RunnersUp (r:1 w:1)
 	// Storage: Council Prime (r:1 w:1)
 	// Storage: Council Proposals (r:1 w:0)
 	// Storage: Council Members (r:0 w:1)
 	fn remove_member_with_replacement() -> Weight {
-		(61_276_000 as Weight)
+		(61_875_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(5 as Weight))
 			.saturating_add(T::DbWeight::get().writes(5 as Weight))
 	}
-	// Storage: PhragmenElection RunnersUp (r:1 w:0)
-	fn remove_member_wrong_refund() -> Weight {
```

Comment:
huh, this got dropped, breaking the build. i'll undo this change

Reply:
Yep not your fault, caused by https://github.com/paritytech/substrate/pull/11790 which removed the benchmark but did not update the weight-trait.

Reply:
@ggwpez should i merge your pr into this branch to fix it rather than readding the old weight?

Reply:
It would be probably slightly cleaner to remove it from the trait as in my MR for Polkadot and Kusama.  
This only makes a difference if someone re-benchmarks on this branch.